lib/repo/git_hooks/max_file_size/.default

```diff
@@ -0,0 +1 @@
+5000000
```

Comment:
You should delete this file (it was leftover from an earlier Docker update, but is no longer necessary).